backends/platform/iphone/iphone_video.mm

```diff
@@ -102,8 +105,10 @@ - (void)createContext {
 	CAEAGLLayer *eaglLayer = (CAEAGLLayer *)self.layer;
 
 	eaglLayer.opaque = YES;
-	eaglLayer.drawableProperties = [NSDictionary dictionaryWithObjectsAndKeys:
-	                                [NSNumber numberWithBool:FALSE], kEAGLDrawablePropertyRetainedBacking, kEAGLColorFormatRGB565, kEAGLDrawablePropertyColorFormat, nil];
+	eaglLayer.drawableProperties = @{
+	                                 kEAGLDrawablePropertyRetainedBacking: @NO,
```

Comment:
This is the « new » Objective-C 2.0 syntax. Which compiler are you using? Anyway, I’ll switch back to the old notation, there is no need to use the new notation, expect for the ease of reading :)